grails-app/services/org/pih/warehouse/inventory/RequisitionStatusTransitionEventService.groovy

```diff
@@ -25,5 +25,6 @@ class RequisitionStatusTransitionEventService implements ApplicationListener<Req
         Requisition requisition = event.requisition
         requisitionService.triggerRequisitionStatusTransition(requisition, event.createdBy)
         notificationService.publishRequisitionStatusTransitionNotifications(requisition)
+        notificationService.publishFulfillmentNotifications(requisition)
```

Comment:
This is just a state transition so it should probably go into the publishRequisitionStatusTransitionNotifications method when we have a chance to refactor.

Reply:
I will move it in my open PR #4298